components/social/org.wso2.carbon.social.core/src/main/java/org/wso2/carbon/social/core/Activity.java

```diff
@@ -41,10 +41,16 @@
 
     int getRating();
 
+    String getComment();
+
 	void setLikeCount(int likeCount);
 
 	void setDislikeCount(int dislikeCount);
 
 	void setId(int id);
 
+	void setRating(int rating);
+
+	void setComment(String comment);
```

Comment:
Thanks @daneshk   , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/c2755ff318863500567b83ba11a3a25bb0a22769)